src/NuGet.Core/NuGet.Packaging/Core/FrameworkNameValidatorUtility.cs

```diff
@@ -34,6 +38,70 @@ internal static bool IsValidFrameworkName(string path)
             return fx != null && fx.Framework != NuGetFramework.UnsupportedFramework.Framework;
         }
 
+        internal static bool IsDottedFrameworkVersion(string path)
```

Comment:
I'd suggest making this method private inside the validation rule.

Firstly, the method name is `IsDottedFrameworkVersion`, but the input is not a framework string. The method is extracting the TFM from a path, and then doing what the name suggests, so it fails the "single responsibility principal", plus the naming convention where a method name should describe what it's doing.

Secondly, it returns "true" when the path doesn't contain a TFM. That's a pretty significant gotcha in case anyone wants to re-use this method in the future.